lib/providers/api/status_api.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:meta/meta.dart';
+import 'package:rxdart/rxdart.dart';
+import 'package:weekplanner/models/giraf_user_model.dart';
+import 'package:weekplanner/models/response_model.dart';
+import 'package:weekplanner/models/role_enum.dart';
+import 'package:weekplanner/providers/http/http.dart';
+import 'package:weekplanner/providers/peristence/persistence.dart';
+
+class StatusApi {
+  final Http _http;
+
+  /// End-point for checking if the API is running.
+  Observable<String> Status() {
+    return _http.get("/Status")
```

Comment:
from `_http.get` we get an `Observable<Response>` and we promised `Observable<bool>`, the following looks at the response from the server and only provides the success boolean

```suggestion
    return _http.get("/").map((Response res) => res.json["success"]);
```